platform/linux-generic/_fdserver.c

```diff
@@ -4,6 +4,10 @@
  * SPDX-License-Identifier:     BSD-3-Clause
  */
 
+#ifdef HAVE_CONFIG_H
```

Comment:
@muvarov. Maybe I shoud just drop that `#ifdef` alltogether, leaving just `#include` in place. I've always just used that as an idiom to allow the same source file to be used no matter what. One can enable or disable header file generation, but the source file would still comple. Nevertheless it does not make sense to put `#ifdef` guarding `config.h` inclusion inside config.h. If there is no config.h (because e.g. it was disabled in configure.ac or because Autoconf/Automake semantics change, for whatever reason), such code will compile. Code with #ifndef inside config.h won't/